microprofile.jdt/com.redhat.microprofile.jdt.quarkus/plugin.xml

```diff
@@ -7,5 +7,9 @@
       <provider class="com.redhat.microprofile.jdt.internal.quarkus.providers.QuarkusConfigPropertiesProvider" />
       <provider class="com.redhat.microprofile.jdt.internal.quarkus.providers.QuarkusKubernetesProvider" />
    </extension>
+   
+   <extension point="com.redhat.microprofile.jdt.core.projectTypes">
+      <type class="com.redhat.microprofile.jdt.internal.quarkus.providers.QuarkusProjectType" label="quarkus"/>
```

Comment:
Move this class in `com.redhat.microprofile.jdt.internal.quarkus` package, this class is not a providers.